.github/ISSUE_TEMPLATE/bug_report_template.yml

```diff
@@ -46,20 +46,12 @@ body:
     id: system-info
     attributes:
       label: System Info
-      description: Please share your system info with us, you can get the Gradio version with `gradio.__version__`
+      description: Please ensure you are running a recent version of Gradio. You can get the Gradio version with `gradio.__version__`
       render: shell
-      placeholder: Gradio version, Operating System, Browser
+      placeholder: 1. Gradio version 2. Operating system 3. (If relevant), the browser
     validations:
       required: true
-  - type: dropdown
-    id: severity
+  - type: markdown
     attributes:
-      label: Severity
-      description: Select the severity of this issue
-      options:
-        - annoying
-        - serious, but I can work around it
-        - blocking upgrade to latest gradio version
-        - blocking all usage of gradio
-    validations:
-      required: true
```

Comment:
I think it would be good to know how bad it is but this could be simplified:

- I can work around it
- blocking use of gradio

Reply:
Ok